CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 vX.Y.Z - TBD
 ------------------------
+Features:
 
-Bug fixes:
+* #125 - Implement CODEOWNERS and auto-assignment to maintainers on PRs
+
+=======
```

Comment:
Hmmm I'm not quite sure what you were going for here, but I don't think this is going to format the way you want it. You can see what this will look like here: https://github.com/godaddy/tartufo/blob/1d4e87219a2a66b4359d207e55ee8ef4af3385f6/CHANGELOG.md

FWIW the way I had laid it out I was just trying to follow what I had done previously, specifically with 2.0.0a1. Not to say it couldn't be done better; but we should try be consistent with however we decide to do it. 😄